### ISSUE TITLE
Stock CM apk`s

### ISSUE THREAD

**Issue by alcarushik** [aymende7/XTutorials_Miui8#2]
At what point to delete stock apk`s of cyanogenmod, like SustemUI....

**Comment by aymende7** [comment]
edit customize_target_files.sh
if you want to delete systemui add this 
`rm -rf out/target_files/SYSTEM/priv-app/SystemUI`

**Comment by XTutorials** [comment]
That would be amazing... still no luck on me, change files in boot.img, even changed the complete boot img and still not even a boot screen.

**Comment by aymende7** [comment]
me too bootloop on 2 devices patchrom hates me

**Comment by XTutorials** [comment]
@alcarushik , do you develop for P8 Lite? Do you have one? I didn't saw you on XDA P8 Lite forum.

**Comment by alcarushik** [comment]
No, i am doing this for LG G3 Beat. I am just do not know where to ask anymore about issues. I have bootloop. Can i delete unnedeed APK`s manually in fullota.zip?

**Comment by aymende7** [comment]
Yes you can.
But this not goona change anything!
First you need to edit boot.IMG to add miui jars to bootclass then enable USB debugger to get the boot log "logcat"

**Comment by alcarushik** [comment]
@aymende7 edit boot.IMG to add miui jars to bootclass - Where can I see the instruction?

**Comment by alcarushik** [comment]
Please, help me with editing boot.img, i`m stuck on bootlopp :(

**Comment by aymende7** [comment]
@alcarushik
 follow only boot.img part https://forum.xda-developers.com/showpost.php?p=69191801&postcount=1234

**Comment by alcarushik** [comment]
Thank you very VERY much!!!! After that editing bootloop again (((( Could you see my log? Please!( https://drive.google.com/open?id=0B60K36fN0dCsRDhnME9vMlU2VjQ

**Comment by aymende7** [comment]
same problem as my rom
there 2 crashes
you can fix 1 crash by removing ResolverProxy.smali & ResolverProxy$ProxyListAdapter.smali from `framework.jar.out/smali/com/android/internal/app`
the other one idk 
i still facing this problem until now :(

**Comment by alcarushik** [comment]
Sad (
About what second issue? About libmmcamera... libs ?

**Comment by aymende7** [comment]
F libc    : Fatal signal 6 (SIGABRT), code -6 in tid 3370 (main)

**Comment by alcarushik** [comment]
omg, It does not mean anything to me )))) so complicated...

**Comment by aymende7** [comment]
me too fatal error causes bootloop :'(

**Comment by alcarushik** [comment]
Hi. Tell me please, if i make patchrom on AOSP based rom, do i have to do "make secondpatch" or enough "make firstpatch"?

**Comment by aymende7** [comment]
secondpatch too
